core/domain/platform_parameter_list.py

```diff
@@ -58,3 +59,9 @@ class PARAM_NAMES(enum.Enum): # pylint: disable=invalid-name
     'This flag is for the end chapter celebration feature.',
     platform_parameter_domain.FeatureStages.PROD,
 )
+
+Registry.create_feature_flag(
```

Comment:
@nithinrdy, @BenHenning has a question. Is this ready to be turned on in prod? Enabling it for prod means that the feature is ready to be turned on in the next prod release of the app (i.e. it's dev complete and has been fully feature tested) -- is this the case?

Also since Ben is quite busy I'll probably take over this review, I'll put a review hold and he'll approve.

Reply:
> @nithinrdy, @BenHenning has a question. Is this ready to be turned on in prod?

No, the feature is yet to be tested.

> Enabling it for prod means that the feature is ready to be turned on in the next prod release of the app (i.e. it's dev complete and has been fully feature tested) -- is this the case?

No this is not the case, the feature can't be enabled for PROD yet as it is yet to undergo a feature review.

However, the feature isn't really enabled for PROD -- it _**can**_ be enabled for PROD because I've put it in the PROD FeatureStage.

The reason I've had to put it in the PROD FeatureStage instead of the DEV FeatureStage is simply because the test server runs in PROD mode. If I place the feature in DEV stage, it can only be enabled in DEV mode, meaning it can't be feature tested on the test server, because the instance of Oppia running on [oppiatestserver.org](oppiatestserver.org) is in PROD mode. For the feature to be 'enable-able' on the test server (oppiatestserver.org), it needs to be in the PROD FeatureStage, following which an admin can enable the feature via the admin page.

> Also since Ben is quite busy I'll probably take over this review, I'll put a review hold and he'll approve.

Got it!

Reply:
Here is @BenHenning's response:

> That's not something I was aware of, and is a good point. Whichever environment QA uses for testing should be set up such that feature flags can be enabled on them when set for the testing environment. It's possible we just never did this last step when implementing feature flags (in which case, PROD is the only current option but this is something you may want to fix for cleaner feature staging).
 
So I think this looks good, @nithinrdy, thanks! I'll approve, and will talk to the release team re fixing this in the future.

Reply:
@seanlip [PTAL](https://docs.google.com/document/d/1FjbG2Cb0OnDVuE36jzux2Sn0sybMQozkk3hjkfeMoC0/edit?pli=1&disco=AAAAacnOBN0) (possibly relevant to this conversation)